src/patternfly/components/AlertGroup/alert-group.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-alert-group{{#if alert-group--modifier}} {{alert-group--modifier}}{{/if}}"
```

Comment:
Is this considered a list? I wonder if we should use a `ul`?

Reply:
Yeah probably. Changed and updated the docs.